js/ui/text_box/ui.text_editor.clear.js

```diff
@@ -0,0 +1,65 @@
+import $ from "../../core/renderer";
+import eventsEngine from "../../events/core/events_engine";
+import ActionButtonBase from "./action_button_collection/button.base";
+import { addNamespace } from "../../events/utils";
+import { down as pointerDown } from "../../events/pointer";
+import { name as click } from "../../events/click";
+
+const STATE_INVISIBLE_CLASS = "dx-state-invisible";
+const TEXTEDITOR_CLEAR_BUTTON_CLASS = "dx-clear-button-area";
+const TEXTEDITOR_CLEAR_ICON_CLASS = "dx-icon-clear";
+const TEXTEDITOR_ICON_CLASS = "dx-icon";
+const TEXTEDITOR_SHOW_CLEAR_BUTTON_CLASS = "dx-show-clear-button";
+
+export default class ClearButton extends ActionButtonBase {
+    constructor(editor, options) {
+        super("clear", editor, options);
+    }
+
+    _createInstance() {
+        const $element = $("<span>")
+            .addClass(TEXTEDITOR_CLEAR_BUTTON_CLASS)
+            .append($("<span>").addClass(TEXTEDITOR_ICON_CLASS).addClass(TEXTEDITOR_CLEAR_ICON_CLASS));
+
+        return {
+            instance: $element,
```

Comment:
Can we provide an object with the interface like a widget instance?
```suggestion
            instance: {
                element: () => $element,
                dispose: () => $element.remove()
            },
```